lib/active_storage_validations/aspect_ratio_validator.rb

```diff
@@ -24,11 +26,12 @@ def validate_each(record, attribute, _value)
         changes = record.attachment_changes[attribute.to_s]
         return true if changes.blank?
 
+        options = unfold_procs(record, self.options, AVAILABLE_CHECKS)

```

Comment:
I don't like changing the original options variable. The `options` variable is made available in the `ActiveModel::Validator` and I would really keep it that way.

Reply:
I hope I understand you correctly but the original `options` variable will not be changed, it will be masked by a local variable called `options`. To avoid this confusion, I'll rename the local variable with `flat_options`.